libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2475,20 +2475,29 @@ string YulUtilFunctions::copyArrayFromStorageToMemoryFunction(ArrayType const& _
 	});
 }
 
-string YulUtilFunctions::bytesConcatFunction(vector<Type const*> const& _argumentTypes)
+string YulUtilFunctions::concatBytesOrStringFunctionCreate(
+	std::string functionName,
+	vector<Type const*> const& _argumentTypes,
+	FunctionType::Kind const& functionTypeKind
```

Comment:
```suggestion
	FunctionType::Kind functionTypeKind
```
(it's a POD, so no need to use a const&)